src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Charts/BarChart/BarChart.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Tooltip,
+  Legend,
+} from 'chart.js';
+import { Bar } from 'react-chartjs-2';
+
+import { variables } from 'shared/styles';
+
+import { mockedSubscales as chartData } from './mock';
+import { getDatasets } from './BarChart.utils';
+import { CustomLegend } from './BarChart.types';
+
+ChartJS.register(CategoryScale, LinearScale, BarElement, Tooltip, Legend);
+
+export const BarChart = () => {
+  const options = {
+    responsive: true,
+    plugins: {
+      legend: {
+        labels: {
+          color: variables.palette.on_surface,
+          font: {
+            family: 'Atkinson',
+            size: 14,
```

Comment:
could you pls use variables.font.size.md?

Reply:
variables.font.size.md is string - '1.4rem'
here should be just a number